google_gke_infra/README.md

```diff
@@ -89,7 +91,9 @@ provider "google-beta" {
 
 ## Usage
 
-### Basic Usage Example
+### Basic configuration
```

Comment:
I thought that `Basic configuration` is a bit better when speaking about how the module is configured. I'm not forcing it though :)